pages/vi/profiles/sidmehta65.md

```diff
@@ -6,7 +6,7 @@
 #### Central daylight time
 #### Windows 8.1
 #### Hello my name is Siddharth Mehta. I am a graduate student at Texas Tech University. I will be graduating in May 2019. I have completed my bachelor's in Computer Engineering from University of Pune.
-![](https://ibb.co/bxvLNS)
+![](https://00e9e64bac7ddcaf8c6289620ae2c9a9158f20db41d4abc6b6-apidata.googleusercontent.com/download/storage/v1/b/sidmehta65github/o/AdminBuilding.jpg?qk=AD5uMEvaHLdkeDR4VXkxj15j6i9BA6k_giTmSu7lm4HTZYhbyyl6VaF3R9a_eMdGgoxo_bs9l-F0z4LNb8_n1X3ZN8k-PTv8aj_uVCK-lVt706ajwUP6aflfN5sWaNvSJ9CuU_ftTzgSF6tNGrd5Y20GfhaJ4U0FpUrR-ngXaZfrXsS0llKFiO_4isWuB9XqdQBUUZwsIwH05WqrZnQ2S6eeozUYhSctK8e1iaNQyvyaEMNPIcQkHsk__pmq5pOBIEQKF0_pu3Ea9ze_GToRDdBfWfkUb4c4dW1_LzgELzohvg8iFyzRR3izyYo2K93wly02GAuO6e6bTidmTFsCYb5gyBbgD03jqm_yFS_shwoude0lu2colx-Pit333h6YyNhkqb4Tm-rer0R1tkXbL0Et-jtgZrZXZoj56iiGVfhKcVdjWx9-hWIsm6ENb1O1aUFCaakMfswQghBvG0GlMoJ9jyeAU58m_HljzdszBo5itUvSNe-OlE6pfs3h5bpwMpwCkE3uqg3h3fQi-oMpssxY_O3_BW-RW3ej4-RdEY_RGVxUfe3Yv4vIYAMsZj3SkBte_PH-NX71d77vwWberEF49zkZX1bCl7eWr1k-8f-VJtOVO1N2vAIIipx6zq1ny2-hd9UA1qCSvjH5xeBH0Av5xo8Hh7eNcHycHQPE81b4R_2MT67kxMWI2gFno1gr4OkFFVJHob700JnoeBvGX4_ZCv00psIv-MKwFehBVJOigjms6MkkMNQ)
```

Comment:
This emded is still giving the error ` {"error":{"errors" [{"domain":"global","reason":"lockedDomainExpired","message":"Locked Domain Expired"}],"code":401,"message":"Locked Domain Expired"}} `

You should remove it and replace it with working link.

Your earlier link `https://ibb.co/bxvLNS` can be used as an embed. you just have to modify it like this `https://preview.ibb.co/mzmS2S/Admin_Building.jpg`

Also please add some alternate text for the image. E.g. `![Admin building]()`